client/layout/guided-tours/tours/jetpack-plugin-updates-tour/index.tsx

```diff
@@ -34,9 +35,26 @@ export const JetpackPluginUpdatesTour = makeTour(
 	>
 		<Step
 			name="init"
-			target=".plugin-item-jetpack .form-toggle__switch"
+			target={ JETPACK_TOGGLE_SELECTOR }
 			arrow="top-left"
 			placement="below"
+			wait={ () =>
+				new Promise( resolve => {
+					if ( document.querySelector( JETPACK_TOGGLE_SELECTOR ) ) {
+						return resolve();
+					}
+
+					const waitForElement = () =>
+						setTimeout( () => {
+							if ( document.querySelector( JETPACK_TOGGLE_SELECTOR ) ) {
+								return resolve();
+							}
+							waitForElement();
```

Comment:
did it cause a problem to do…

```js
const waitForElement = () => {
	if ( … ) {
		return resole();
	}

	setTimeout( waitForElement, 125 );
}
```

more or less curious because we shouldn't need to wrap this the way we have…even though neither is "right" or "wrong"

Reply:
Nice, applied and working well 👍